api/db/base.js

```diff
@@ -62,7 +61,9 @@ const defaultSyncChildren = async (
 // These properties are added to model instances.
 const instanceExtension = (orm, models) => ({
   pickUpdateable(raw) {
-    return pick(raw, this.static.updateableFields);
+    return Object.entries(raw)
+      .filter(([key]) => this.static.updateableFields.includes(key))
+      .reduce((obj, [key, value]) => ({ ...obj, [key]: value }), {});
```

Comment:
I don't really understand what this change does/why this change exists, but I'm assuming it has something to do with updating something...

Reply:
Just getting rid of `lodash.pick` as a dependency.  It drags in a bunch of code (it handles a lot more use cases than we need) and the lodash family of packages are pretty routinely coming up in Snyk vulnerability reports.  So...  I figured just get rid of it.  😛